src/iuse.cpp

```diff
@@ -3004,7 +3004,7 @@ int iuse::pickaxe(player *p, item *it, bool, const tripoint& )
         ( g->m.has_flag( "SUPPORTS_ROOF", dirx, diry ) || g->m.has_flag( "MINEABLE", dirx, diry ) ) ) {
 
         /** @EFFECT_STR speeds up mining with a pickaxe */
-        turns = ( ( MAX_STAT + 4 ) - std::min( p->str_cur, MAX_STAT ) ) * MINUTES( 5 );
+        turns = ( ( MAX_STAT + 4 ) - std::min( p->str_cur, MAX_STAT ) ) * MINUTES( 5 ) * 100;
```

Comment:
Seems this variable is misnamed. Apparently is stores the amount of *moves*, not of turns. Either change its name, or apply the conversion from turns to moves when calling `player::assign_activity` and change the 20000 to `MINUTES( 20 )`.

Reply:
> and change the 20000 to MINUTES( 20 ).

Shouldn't I change it to `20_minutes` instead?